lib/_utils/constants/routes.dart

```diff
@@ -8,13 +9,15 @@ class RouteNames {
   static const String onboarding = "/onboarding";
   static const String mainHome = "/main_home";
   static const String homePage = "/home_page";
+  static const String dashboard = "/dashboard";
 }
 
 final publicPages = [
   GetPage(name: '/', page: () => const Welcome()),
-  GetPage(name: '/', page: () => const LoginPageScreen()),
+  GetPage(name: '/login', page: () => const Login()),
```

Comment:
Put **RouteNames.loginPage** in param **name**

Reply:
Resolved by adding  the defined route names.